SpatialGDK/Source/SpatialGDKServices/Private/SSpatialOutputLog.cpp

```diff
@@ -313,38 +292,33 @@ void SSpatialOutputLog::FormatAndPrintRawLogLine(const FString& LogLine)
 		FormatAndPrintRawErrorLine(LogLine);
 		return;
 	}
+	FString LogCategory = LogMatcher.GetCaptureGroup(1);
+	FString LogLevelText = LogMatcher.GetCaptureGroup(2);
+	FString LogMessage = LogMatcher.GetCaptureGroup(3);
 
-	FString LogLevelText = LogMatcher.GetCaptureGroup(1);
-	FString LogMessage = LogMatcher.GetCaptureGroup(2);
-	FString LogCategory = LogMatcher.GetCaptureGroup(4);
+	const FRegexPattern WorkerLogPattern = FRegexPattern(TEXT("\\[(.*)\\] (.*)"));
```

Comment:
Same here, please give an example of what the regex is looking for as it was before. i.e. [UnrealWorkerF5C56488482FEDC37B10E382770067E3:UnrealWorker]